src/bootstrap.php

```diff
@@ -155,9 +155,14 @@
         $digests[$json['id']] = $json;
     }
 
-    uasort($digests, function (array $a, array $b) {
-        return DateTimeImmutable::createFromFormat(DATE_ATOM,
-                $b['published']) <=> DateTimeImmutable::createFromFormat(DATE_ATOM, $a['published']);
+    $dateFactory = function ($item) {
+        return DateTimeImmutable::createFromFormat(
+            DATE_ATOM,
+            $item['published'] ?? '2038-01-01T00:00:00Z'
```

Comment:
Long enough in the future to not be a problem. Could always create an instance dynamically (eg `new DateTimeImmutable('+1 year')` or something).

Reply:
The other dates are not dynamic though so I'd keep it simple. May revisit if we decide on an explicit order also based on `stage`